index.bs

```diff
@@ -106,6 +109,8 @@ The following example shows an image that is registered for observation via its
 The following are sample elements whose rendering timestamps could be measured by using this API and which should be compared to page navigation:
 * The images in the image carousel of a shopping site.
 * The main photo in a story of a news site.
+* The title of a blog post.
+* The first paragraph in an entry of an encyclopedia site.
```

Comment:
Shouldn't this be the largest paragraph, rather than the first?

Reply:
I think first works: it is the first one that a user will attempt to read, so rendering it fast is particularly important.

Reply:
yeah, first works. Please ignore :)